packages/next/taskfile.js

```diff
@@ -176,6 +176,47 @@ export async function ncc_node_fetch(task, opts) {
     .target('compiled/node-fetch')
 }
 
+// eslint-disable-next-line camelcase
+export async function compile_config_schema(task, opts) {
+  const { configSchema } = require('./dist/server/config-schema')
+  // eslint-disable-next-line
+  const Ajv = require('ajv')
+  // eslint-disable-next-line
+  const standaloneCode = require('ajv/dist/standalone').default
+  // eslint-disable-next-line
+  const ajv = new Ajv({ code: { source: true }, allErrors: true })
+  ajv.addKeyword({
```

Comment:
Can you explain why a keyword is being added?

Reply:
It's to allow validating the function config type e.g. `headers()`, `redirects()`, and `rewrites()`